### ISSUE TITLE
ipyvolume example gives an error

### ISSUE THREAD

**Issue by robmarkcole** [QuantStack/voila#6]
```
[Voila] Executing notebook with kernel: python3
WARNING:tornado.access:404 GET /custom.css (::1) 0.85ms
WARNING:tornado.access:404 GET /custom.css (::1) 0.32ms
[Voila] Adapting to protocol v5.1 for kernel 26a4a1ff-c3c3-406c-b1f3-b24dba6afd04
[IPKernelApp] WARNING | Unknown message type: 'custom_message'
WARNING:tornado.access:404 GET /static/dist/jupyter-threejs.js (::1) 0.58ms
WARNING:tornado.access:404 GET /static/dist/ipyvolume.js (::1) 1.13ms
```

**Comment by maartenbreddels** [comment]
ping @SylvainCorlay @vidartf
I wonder if it is related to:
https://github.com/jupyter-widgets/pythreejs/issues/171 which got fixed by this workaround https://github.com/jupyter-widgets/ipywidgets/pull/2026

**Comment by SylvainCorlay** [comment]
Fixed in PR #12 using @vidarf's fix from ipywidgets! Thanks for the pointer.

**Comment by maartenbreddels** [comment]
Great, that will also make ipysheet work!
